@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -502,20 +495,31 @@ function placePointLabelChoosingAnchor(
 
         // Glyphs arrangement have been changed remove text buffer object and apply
         // new text layout to the label.
-        if (i !== 0) {
+        if (!recentPlacementTest) {
```

Comment:
Move this invalidation of label.textBufferObject together with the code that makes the object invalid, i.e. next to the call to textCanvas.measureText.

Reply:
Ok moved them to placePointLabelAtAnchor() function, so the logic works the same and the flag `forceInvalidation` which equals `!isLastPlacement` is used to determine if buffer should be released. This will never happen if we use the last rendered text position.